scripts/build.py

```diff
@@ -156,29 +226,99 @@ def process_js(source_path, target_path):
     _minify(source_path, target_path)
 
 
-def build_minified_third_party_libs(output_directory):
+def get_dependency_filepaths(dependency, filepaths):
+    """Extracts dependency filepaths into filepaths object.
+
+    Args:
+        source_paths: dict(str, str). Dependency information object.
+        filepaths: dict(str, list(str)). Filepaths object with three lists each
```

Comment:
Ditto: what are the possible keys, what are the descriptions and types of their values?

I'd write something like this (please check that it's correct):

```
   filepaths: dict(str, list(str)). A dict mapping file types to lists of filepaths. The dict has three keys: 'js', 'css' and 'fonts'. Each of the corresponding values is a full list of dependency file paths of the given type.
```

Reply:
Done.